Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -376,7 +376,7 @@
     <string name="enable_cpu_cache_description">Enables emulation of the CPU write-back cache. Enabling will have a significant impact on performance. This should be left disabled unless absolutely needed.</string>
     <string name="clock_override">Clock Override</string>
     <string name="overclock_enable">Override Emulated CPU Clock Speed</string>
-    <string name="overclock_enable_description">Higher values can make variable-framerate games run at a higher framerate, requiring a powerful device. Lower values make games run at a lower framerate, increasing emulation speed, but reducing the emulated console\'s performance.</string>
+    <string name="overclock_enable_description">Higher values can make games run at a higher frame rate if they lagged on original hardware, requiring a powerful device. Lower values improve performance by causing lag, but will slow down the game (without affecting audio) if its speed is tied to the frame rate.</string>
```

Comment:
The first part is technically wrong. Higher values can make games run at a higher framerate if they lag in Dolphin. Dolphin's CPU timings aren't good enough to accurately emulate whether games lag on original hardware.

For the second part, I think users could get quite confused about how causing lag improves performance. The older wording of running at a lower framerate was clearer in that aspect.

Reply:
Is this better?
I'm still not using the old description because I think that it may confuse a user on the difference between this and VBI Frequency.

Reply:
Yes, it's better.